Translators/WZDx/main.py

```diff
@@ -75,6 +75,66 @@ def translate(wzdx_geojson):
             logging.info(f'Failed to generate TIM for feature: {feature["id"]}')
     return tims
 
+def translate(wzdx_geojson):

```

Comment:
(blocking) Are there any unit tests for this method at the moment? It would be nice to be able to view an example of the expected input & output of this method.

Reply:
There were not - I've moved this method to tim_translator.py and have added tests for it